docs/README.md

```diff
@@ -1,18 +1,19 @@
-[KroxyDoc]: https://kroxylicious.io/ "Kroxylicious documentation"
+[KroxyliciousDoc]: https://kroxylicious.io/ "Kroxylicious documentation"
 [AsciiDoc]: https://docs.asciidoctor.org/asciidoc/latest/syntax-quick-reference/  "AsciiDoc reference"
 
 <!-- omit from toc -->
 # Kroxylicious documentation
 
 Welcome to the Kroxylicious documentation! 
 
-This folder contains the source files for managing the Kroxylicious documentation. The documentation is written in [AsciiDoc][AsciiDoc] and generated using 
- and it provides content to help you understand and set up Kroxylicious proxies.
+This folder contains the source files for managing the Kroxylicious documentation. 
+The documentation is written in [AsciiDoc][AsciiDoc] and provides content to help you understand and set up Kroxylicious proxies.
```

Comment:
we've been generally referring to the component as simply "Kroxylicious" rather than "Kroxylicious proxy".
I think the existing ref was an exception rather than a rule.

```suggestion
The documentation is written in [AsciiDoc][AsciiDoc] and provides content to help you understand and set up Kroxylicious proxies.
```